pkg/acceptance/cluster/dockercluster.go

```diff
@@ -538,6 +538,10 @@ func (l *DockerCluster) processEvent(ctx context.Context, event events.Message)
 	l.mu.Lock()
 	defer l.mu.Unlock()
 
+	// Logging everything we get from Docker in service of finding the root
+	// cause of #58955.
+	log.Infof(ctx, "processing event from Docker: %s", event)
```

Comment:
are you sure you want `%s` here and not `%+v`?

Reply:
Fixed :)